.teamcity/_self/projects/WebApp.kt

```diff
@@ -733,8 +733,8 @@ object CheckCodeStyleBranch : BuildType({
 	}
 })
 
-object RunCalypsoPlaywrightE2eTests : BuildType({
-	name = "Playwright E2E tests"
+object RunCalypsoPlaywrightE2eDesktopTests : BuildType({
+	name = "Playwright E2E tests (desktop)"
```

Comment:
TODO **after** this is merged: we should update the BuildType with the `uuid` generated by TC, so we can eventually rename it (it won't make sense to have it named `Playwright E2E tests (desktop)` once we delete `E2E tests (desktop)`

Reply:
Filed task #53731 for the todo item.